xarray/core/dataarray.py

```diff
@@ -7097,3 +7097,13 @@ def to_dask_dataframe(
     # this needs to be at the end, or mypy will confuse with `str`
     # https://mypy.readthedocs.io/en/latest/common_issues.html#dealing-with-conflicting-names
     str = utils.UncachedAccessor(StringAccessor["DataArray"])
+
+    def drop_attrs(self) -> Self:
+        """
+        Removes all attributes from the DataArray.
+        """
+        self = self.copy()
+
+        self.attrs = {}
```

Comment:
I think the coords can have their own attrs?

Reply:
Those would get picked up in `for var in self.variables:`, right?

Reply:
`for var in self._coords.values()`, yes

Reply:
I had meant — they already get picked up with the proposed code, since it contains `for var in self.variables` — no? I added a test for this, lmk where you think that's missing, very possible I'm missing something...